### ISSUE TITLE
Distributed TensorFlow Mask R-CNN Notebooks needs to use default values in all cells

### ISSUE THREAD

**Issue by ajayvohra2005** [aws/amazon-sagemaker-examples#4436]
**Link to the notebook**
Add the link to the notebook. https://github.com/aws/amazon-sagemaker-examples/blob/main/advanced_functionality/distributed_tensorflow_mask_rcnn/mask-rcnn-scriptmode-experiment-trials.ipynb

**What aspects of the notebook can be improved?**
Notebook needs to use default set of values in all cells so running all cells in CI/CD testing completes successfully. Also, BYOC containers need to be updated to TensorFlow 2.12.0.

**What are your suggestions?**
1. Use SageMaker S3 bucket when no S3 bucket is specified.
2. Set default model type
3. Build and push default training image
4. Specify default training script
5. Update containers to use TensorFlow 2.12.0

**Comment by ajayvohra2005** [comment]
Will fix CI build errors and open a new PR.
